Kwartracker/Scenes/Reusable/TextGroupView.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  TextGroupView.swift
+//  Kwartracker
+//
+//  Created by Mickale Saturre on 5/31/21.
+//
+
+import SwiftUI
+
+struct TextGroupView: View {
+    let text: String
+    let fontSize: CGFloat?
+    let bottomMargin: CGFloat?
+    let fontColor: Color
+    
+    private let defaultFontSize: CGFloat = 17
+    private let defaultBottomMargin: CGFloat = 15
```

Comment:
```suggestion
init(
        text: String,
        fontSize: CGFloat = 17,
        bottomMargin: CGFloat = 15,
        fontColor: Color
    ) {
        self.text = text
        self.fontSize = fontSize
        self.bottomMargin = bottomMargin
        self.fontColor = fontColor
    }
```

Reply:
I tried doing this but it give me an error https://github.com/jaimejazarenoiii/kwartracker-ios/pull/35/files#r643682589